internals/overlord/checkstate/manager.go

```diff
@@ -349,10 +375,75 @@ type CheckInfo struct {
 type CheckStatus string
 
 const (
-	CheckStatusUp   CheckStatus = "up"
-	CheckStatusDown CheckStatus = "down"
+	CheckStatusUp       CheckStatus = "up"
+	CheckStatusDown     CheckStatus = "down"
+	CheckStatusInactive CheckStatus = "inactive"
 )
 
 type checker interface {
 	check(ctx context.Context) error
 }
+
+// StartChecks starts the specified checks by creating a new perform-checks
+// change for each check that is in the current plan and not already running.
```

Comment:
I think we should probably drop the "by creating a new perform-checks change" (seems too much like implementation details), and add what the return value represents.